src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -331,24 +331,493 @@ The following workflows represent some mathematical functions that can be optimi
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `DOCKER_IMAGE`
+| the Docker image that contains all the dependencies and libraries needed for the model execution.
+| String (default="activeeon/dlm3")
+| `INSTANCE_NAME`
+| the name of the service that will be deployed.
+| String (default="model-server")
+| `YAML_FILE`
+| A YAML file that describes the OpenAPI Specification (known as Swagger Spec) of the service. This file should be stored in the catalog under the `model_as_service_resources` bucket. This file should be stored in the Catalog. More information about the structure of this file can be found in the section <<Customize the Service>>.
+| String (default="ml_service-api").
+| `PYTHON_ENTRYPOINT`
+| This entry script starts the service and defines the different functions to deploy the model, score the requests by using the model, and return the results. This specific script is specific to your model. This file should be stored in the Catalog. More information about this file can be found in the <<Customize the Service>> section.
+| String (default="ml_service").
+|===
+
+Click on `Execute Action` and follow the progress of the service creation.
+
+image::MAS_Model_Servie_PCA.png[500,500,align=center]
+
+Open the Endpoint and concatenate the base url `api/ui`to your endpoint.
+
+Through this Swagger api, you are now able to :
+
+** access the different defined endpoints
+** ask for an api_token
+** deploy a model
+** score some data based on the deployed model
+
+Each generated endpoint will be explained in the next subsections.
+
+*Via Studio portal*
+
+Open the link:https://try.activeeon.com/Studio[Studio Portal].
+
+Create a new workflow.
+
+Add the `model_as_a_service` bucket.
+
+Drag and drop the `Model_Service_Start` task from the bucket.
+
+Execute the Workflow by setting the different workflow variables as following:
+
+.Model_Service_Start variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `DOCKER_IMAGE`
+| the Docker image that contains all the dependencies and libraries needed for the model execution.
+| String (default="activeeon/dlm3")
+| `INSTANCE_NAME`
+| the name of the service that will be deployed.
+| String (default="model-server")
+| `YAML_FILE`
+| A YAML file that describes the OpenAPI Specification (known as Swagger Spec) of the service. This file should be stored in the Catalog under the `model_as_service_resources` bucket. More information about the structure of this file can be found in the section <<Customize the Service>>.
+| String (default="ml_service-api").
+| `PYTHON_ENTRYPOINT`
+| This entry script starts the service and defines the different functions to deploy the model, score the requests by using the model, and return the results. This specific script is specific to your model. This file should be stored in the Catalog under the `model_as_service_resources` bucket. More information about this file can be found in the <<Customize the Service>> section.
```

Comment:
`This specific script is specific to your model.` . => `This script is specific to your model.`

Reply:
ok